### DISCUSSION TITLE
metadata update

### DISCUSSION THREAD

**Discussion by happy1229** [Difegue/LANraragi#776] [Q&A] [answered]
Hi
Is there a way to quickly update metadata
For example, I downloaded 12345.zip containing info.json in 2020
But in 2023 I downloaded the 12345.zip again but there is a difference
The internal info.json has changed and there are fewer tags
But after using Plugin again, the metadata will not be less at the same time

**Comment by Difegue** [comment]
👋 Metadata plugins don't have a way to remove/replace tags for now, it's all additive.  
I'd recommend clearing out the metadata in the edit page and then re-running the plugin.

**Comment by Difegue** [comment, marked as answer]
I only thought about this now when compiling release notes, but with #759, you should now be able to just copy your archives out of the content folder and add them again with the "Add archive" web uploader, which with this option enabled should overwrite the existing data, metadata included.  

It's a bit clunky..but it works! 👍
